crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -451,6 +457,28 @@ def delete_pacemaker_resources(node_name)
         end
       end
 
+      # Evacuate all routers away from the specified network node to other
+      # available network nodes. The evacuation procedure it started on the
+      # specified controller node
+      def evacuate_network_node(controller, network_node, delete_namespaces = false)
+        args = [network_node]
+        args << "delete-ns" if delete_namespaces
+        controller.wait_for_script_to_finish(
+          "/usr/sbin/crowbar-router-migration.sh", 600, args
```

Comment:
This hardwired `600` looks strange, although I understand that this was the timeout originally for the other script. Given that we are using `--wait-for-router`, execution time can increase hugely. I am not sure we need to address it here, but at some point this hardwired value could be a real pain. Best would be to make the timeout's value depending on the number of routers. This is something that we might want to put inside neutron-ha-tool